### ISSUE TITLE
ls -l option fails when it encounters non-ascii characters in the object name

### ISSUE THREAD

**Issue by jterrace** [GoogleCloudPlatform/gsutil#85]
_Original author: differen...@gmail.com (March 16, 2012 03:15:50)_

<b>What steps will reproduce the problem?</b>
1. gsutil ls -l of bucket having an object containing non-ascii, specifically CJK characters
2. pipe the output to a file
3. STDERR will have the error message indicating failure

What is the expected output? 
Expected output is a complete listing of all objects in the bucket.

What do you see instead?
Incomplete output along with a message like this-
Failure: 'ascii' codec can't encode characters in position 106-109: ordinal not in range(128).

What version of the product are you using? 
2.0

On what operating system?
CentOS 5

<b>Please provide any additional information below.</b>

Characters I allow in the object names are in this range:
\x{00c0}-\x{0233}
\x{3000}-\x{303f}
\x{3040}-\x{309f}
\x{30a0}-\x{30ff}
\x{4e00}-\x{9faf}
\x{ff00}-\x{ffef}

basically CJK characters

_Original issue: http://code.google.com/p/gsutil/issues/detail?id=86_
